Cargo.toml

```diff
@@ -36,11 +36,17 @@ tempfile = "3.8"
 arrow2 = "0.18"
 
 [workspace]
-members = ["gdal-sys"]
+members = ["gdal-src", "gdal-sys"]
 
 # docs.rs-specific configuration
 [package.metadata.docs.rs]
 # include `array` feature in documentation
 features = ["array"]
 # define attribute `docsrs` for feature badges
 rustdoc-args = ["--cfg", "docsrs"]
+
+
+[patch.crates-io]
+proj-sys = { git = "https://github.com/GiGainfosystems/proj", rev = "dc67b5afc17bc7d330371e4406dffd65cf9321cd" }
+libsqlite3-sys = { git = "https://github.com/rusqlite/rusqlite/", rev = "42a82fb" }
+pq-sys = { git = "https://github.com/sgrif/pq-sys", rev = "77e6729" }
```

Comment:
This requires https://github.com/sgrif/pq-sys/pull/50 

(I will likely merge this and cut a `0.1.1` release in the next days. Afterwards I will update this PR)

Reply:
I've cut a 0.1.1 release of pq-sys and removed the patch section here afterwards.